cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinator.scala

```diff
@@ -101,11 +108,11 @@ private[akka] final class BootstrapCoordinator(discovery: SimpleServiceDiscovery
 
   import BootstrapCoordinator.Protocol._
   import BootstrapCoordinator._
-  import context.dispatcher
 
+  implicit private val ec = context.dispatcher
```

Comment:
changed because of https://github.com/akka/akka/issues/25040 ?

Reply:
yeah, as general style recommendation rather than an issue that I encountered here tho

Reply:
yeah :+1: